config/config.go

```diff
@@ -116,13 +116,13 @@ var baseConfig = Config{
 		EvictionRatio:          0.9,
 		KeysLimit:              200000000,
```

Comment:
KeysLimit is also getting used at `internal/store` Shall we keep this value small as well by default? We don't need these many keys to consider for eviction. 

Shall we keep KeysLimit and WatchChanBufSize as two different variables? Hope there's no logical correlation between them

Reply:
Reduced default value to 20M from 200M, `KeysLimit:              20000000,`
`WatchChanBufSize` is a separate config var created already.